.yarnrc.yml

```diff
@@ -0,0 +1 @@
+npmRegistryServer: "https://registry.npmjs.org/"
```

Comment:
Adding this file ensures that our yarn.lock file doesn't accidentally start containing dependencies from salesforce proxy servers.